src/utils/apiRequests/api.ts

```diff
@@ -159,6 +162,7 @@ export async function getAuthenticatedRequest<T>(
 }
 
 export async function postAuthenticatedRequest<T>(
+  email?: string,
   url: any,
```

Comment:
@sunilsabatp `email` is an optional parameter and should appear later in the order. Otherwise, we'd need to pass `undefined` in every call which is unnecessary and makes the code harder to read.